setup/package.dfsrc

```diff
@@ -1,6 +1,6 @@
 # Please note that this Dockerfile is used for testing nightly builds and should
 # not be used to deploy Odoo
-FROM debian:bookworm
+FROM ubuntu:noble
```

Comment:
Why in this PR?
I would replace bookworm by trixie.

Reply:
We just met with moc, he is gonna do those changes in 19.0 in a dedicated PR. We'll hold this PR until we can rebase :)